lib/compat/wordpress-6.3/script-loader.php

```diff
@@ -42,18 +42,39 @@ function _gutenberg_get_iframed_editor_assets() {
 	$wp_styles->registered  = $current_wp_styles->registered;
 	$wp_scripts->registered = $current_wp_scripts->registered;
 
-	wp_enqueue_style( 'wp-block-editor-content' );
-	// To do: investigate why this is not enqueued through enqueue_block_assets,
-	// as styles for non-core blocks are.
-	wp_enqueue_style( 'wp-block-library' );
+	// We do not need reset styles for the iframed editor.
+	$wp_styles->done = array( 'wp-reset-editor-styles' );
+
 	wp_enqueue_script( 'wp-polyfill' );
+	// Enqueue the `editorStyle` handles for all core block, and dependencies.
+	wp_enqueue_style( 'wp-edit-blocks' );
```

Comment:
Documenting what I see. 

`wp-edit-blocks` corresponds with the editor styles of the core blocks ([core](https://github.com/WordPress/wordpress-develop/blob/44038873e242ae10ea2c20187a431feaae166ac8/src/wp-includes/script-loader.php#L1651), [gutenberg](https://github.com/WordPress/gutenberg/blob/85704f4bd051908bdd8c0ff66306d7831d6c60c9/lib/client-assets.php#L359)).  `wp-block-editor-content` and `wp-block-library` are dependencies of it, so no need to enqueue them explicity, hence we remove them.

By marking `wp-reset-editor-styles` as "done", they won't be enqueue despite also being a dependency.